app/views/application/_navbar.html.erb

```diff
@@ -0,0 +1,8 @@
+<nav>
+<h3><%= link_to t("application.home"), root_path %> |
+<%= link_to t("application.signup"), new_user_path %> |
+<% if current_user %>
+  <%= link_to t("application.signout"), session_path(current_user), method: :delete %> |
+<% end %>
+<%= link_to t("application.signin"), login_path %></h3>
```

Comment:
You probably want to include this link if there is no current_user. So add an `else` branch`.
i.e. if current user, then show to logout link, otherwise, show the login, and sign up links.

Reply:
That's fair and makes sense